.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "3rdParty/spdlog"]
 	path = 3rdParty/spdlog
 	url = https://github.com/gabime/spdlog.git
+	branch = v1.4.2
```

Comment:
1.  The change is not compiling. It seems that spdlog was not updated in CANdb repository. I think you should cd to 3rdParty/spdlog set requested SHA/branch and then commit the change as a CANdb commit.

2. It seems that your changes does not compile with branch v1.4.2? I had to use master (a51b4856377a71f81b6d74b9af459305c4c644f8) to make it compile

Reply:
The problem is with `from_str` that converts log level from string. This is a convenience function that we could replace with hand written one (until it's is the release). 
I always thought that spdlog should be optionally enabled; I'll work on that